serverless/create-record/index.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+/*
+*
+*   Title : Metrics collection for Covid Alert
+*   Purpose : Receive a json file from web or mobile app.
+*   Author : Timothy Patrick Jodoin
+*   Title : Principal Solution Architect
+*   Email : tj@fsdcsolutions.com
+*   Firm : FSDC Solutions Inc.
+*   Date : Jan 12th, 2021
+*   Client : CDS
+*
+*/
+
+const
+    AWS = require( 'aws-sdk' ),
+    uuid = require( 'uuid' ),
+    S3  = new AWS.S3(),
+    fs = require( 'fs' );
+
+exports.handler = async (event, context) => {
+
+    let errorMessage = "Malformed Package";
+    let transactionStatus = "FAILED";
+    let bucket = process.env.dataBucket;           //encrypted env_var
+    let fP = process.env.fileLoca;                  //encrypted env_var
+    let d = new Date();
+    let filename = uuid.v1();  // RFC4122 standard uuid Timestamp based generation
+
+
+// informational for DEV
```

Comment:
inline comments as such are not useful, lets keep the code as clean as possible (and readable)